bucket/flow-launcher.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "1.9.1",
+    "description": "Quick file search & app launcher for Windows with community-made plugins",
+    "homepage": "https://www.flowlauncher.com",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Flow-Launcher/Flow.Launcher/releases/download/v1.9.1/Flow-Launcher-Portable.zip",
+            "hash": "c2560b0ed1d157e4ad6aab0193404583c79f9f50e08b44061cf1f998e01a7ef2"
+        }
+    },
+    "extract_dir": "FlowLauncher",
+    "bin": "Flow.Launcher.exe",
+    "shortcuts": [
+        [
+            "Flow.Launcher.exe",
+            "Flow Launcher"
+        ]
+    ],
+    "persist": [
+        "UserData",
```

Comment:
It would be helpful if you clarify this part too.

Reply:
> Could you explain why there are 2 entries here?
It was made by my while referring to the [wiki](https://github.com/ScoopInstaller/Scoop/wiki/Persistent-data) for persistent data
The Folder `app-1.9.1` contains Userdata folder which contains all settings
however since it changes every update so after referring to the wiki thats how i wrote it 

Is there another way to achieve the same ?

Reply:
> Could you explain why there are 2 entries here?

Hey @rashil2000 
It was made by my while referring to the [wiki](https://github.com/ScoopInstaller/Scoop/wiki/Persistent-data) for persistent data
The Folder `app-1.9.1` contains Userdata folder which contains all settings
however since it changes every update so after referring to the wiki thats how i wrote it 

In case its wrong 
Is there another way to achieve the same ?

Reply:
You've put 2 entries here. I think you want only the second one?

![image](https://user-images.githubusercontent.com/46838874/149170276-54f27623-a318-441a-8226-39d7af7e5bd9.png)

Reply:
Also, select a different name for the target, as UserData might be confusing.

Reply:
@rashil2000 thank you for telling, but is this what you mean ?

```json
    "persist": [
        [
            "app-1.9.1\\UserData",
            "UserData"
        ]
    ],
```
Ill test this later today because i currently dont have access to my pc

Also i feel like UserData is fine for the target unless you have a specific suggestion ?

Thanks

Reply:
Corrected.

Reply:
Why did you remove persist from autoupdate

Reply:
Please keep the persist section in autoupdate so that when the version number increases the persist folder name will also update.

Reply:
Removed because it updated the version fine and changed user settings are still kept. Sounds like it is still required, I will added back.